src/vivarium_nih_us_cvd/configuration/causes.yaml

```diff
@@ -79,40 +86,52 @@ causes:
                     disability_weight: "cause.acute_myocardial_infarction.disability_weight"
                     excess_mortality_rate: "cause.acute_myocardial_infarction.excess_mortality_rate"
         transitions:
-            acute_mi_incidence:
+            susceptible_transient_incidence:
                 source: "susceptible"
-                sink: "acute_myocardial_infarction"
+                sink: "transient_susceptible_state"

```

Comment:
is this effectively defining a self-transition?

Reply:
The transient state (called the transition state) is described [here](https://vivarium-research.readthedocs.io/en/latest/models/causes/ischemic_heart_disease/gbd_2019/index.html#mi-and-hf).